scripts/postinstall.js

```diff
@@ -0,0 +1,31 @@
+const chalk = require('chalk')
+const fs = require('fs-extra')
+
+// proccess.env.PWD is undifined on Windows https://github.com/mrblueblue/gettext-loader/issues/18
+const pwd = process.cwd();
+const destDir = `${pwd}/../@types/nexus-plugin-prisma`
```

Comment:
Surprised this works for windows, it doesn't need to be:

```suggestion
const destDir = `path.join(pwd, '..', '@types', 'nexus-plugin-prisma')`
```

?